src/P/Bool.hs

```diff
@@ -34,3 +36,13 @@ ifM p x y =
 
 guardM :: MonadPlus m => m Bool -> m ()
 guardM f = guard =<< f
+
+-- | Logical disjunction.
+orP :: (a -> Bool) -> (a -> Bool) -> a -> Bool
+p `orP` q = \x -> (p x) || (q x)
+
+-- | Logical conjunction.
+andP :: (a -> Bool) -> (a -> Bool) -> a -> Bool
+p `andP` q = \x -> (p x) && (q x)
```

Comment:
You could also have more general type signatures, I think this would be better:
```hs
-- | Logical disjunction.
orA :: Applicative f => f Bool -> f Bool -> f Bool
orA = liftA2 (||)

-- | Logical conjunction.
andA :: Applicative f => f Bool -> f Bool -> f Bool
andA = liftA2 (&&)
```